website/docs/r/s3_object.html.markdown

```diff
@@ -134,6 +134,8 @@ resource "aws_s3_object" "examplebucket_object" {
 S3 objects support a [maximum of 10 tags](https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-tagging.html).
 If the resource's own `tags` and the provider-level `default_tags` would together lead to more than 10 tags on an S3 object, use the `override_provider` configuration block to suppress any provider-level `default_tags`.
 
+-> S3 objects stored in Amazon S3 Express directory buckets do not support tags, so any provider-level `default_tags` must be ignored.
```

Comment:
```suggestion
-> S3 objects stored in Amazon S3 Express directory buckets do not support tags, so any provider-level `default_tags` will be ignored.
```

Reply:
I think I actually me _must be suppressed_ as the block of text above is talking about -- I'll address in a follow-up, issue/PR.